image.go

```diff
@@ -708,6 +708,46 @@ func (i *Image) At(x, y int) color.Color {
 	return color.RGBA{pix[0], pix[1], pix[2], pix[3]}
 }
 
+// RGBA64At returns the RGBA64 color of the image at (x, y). It is equivalent
+// to calling At(x, y).RGBA() and converting the resulting 32-bit return values to
+// a color.RGBA64, but it can avoid allocations from converting between color types.
```

Comment:
This paragraph is just copied from the standard library's, right? I think this first paragraph should be enough:

```
RGBA64At implements image.RGBA64Image's RGBA64At.
```